File: dev_android_crie_18_apps/Componentesdeinterface2/app/src/main/java/com/example/componentesdeinterface2/MainActivity.java
package com.example.componentesdeinterface2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.CheckBox;
import android.widget.Switch;
import android.widget.TextView;
import android.widget.ToggleButton;

public class MainActivity extends AppCompatActivity {

    private Switch switchEstado;
    private ToggleButton toggleEstado;
    private CheckBox checkEstado;
    private TextView textResultado;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        switchEstado = findViewById(R.id.switchEstado);
        toggleEstado = findViewById(R.id.toggleEstado);
        checkEstado = findViewById(R.id.checkEstado);
        textResultado = findViewById(R.id.textResultado);
    }

    public void enviar(View view){
        textResultado.setText("");

        if(switchEstado.isChecked()){
            textResultado.append("Switch ligado\n");
        }else if(!switchEstado.isChecked()){
            textResultado.append("Switch desligado\n");
        }

        if(toggleEstado.isChecked()){
            textResultado.append("Toggle ligado\n");
        }else if(!toggleEstado.isChecked()){
            textResultado.append("Toggle desligado\n");
        }

        if(checkEstado.isChecked()){
            textResultado.append("CheckBox ligado\n");
        }else if(!checkEstado.isChecked()){
            textResultado.append("CheckBox desligado\n");
        }
    }
}

File: curso_dev_kotlin/Motivation/app/src/main/java/com/example/motivation/ui/MainActivity.kt
package com.example.motivation.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.example.motivation.infra.MotivationConstants
import com.example.motivation.R
import com.example.motivation.infra.SecurityPreferences
import com.example.motivation.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity(), View.OnClickListener {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        // Esconde barra superior
        supportActionBar?.hide()

        handleUserName()

        // Eventos
        binding.buttonNewPhrase.setOnClickListener(this)


    }

    override fun onClick(view: View) {
        if (view.id == R.id.button_new_phrase){
            var s = ""
        }
    }

    private fun handleUserName(){
        val name = SecurityPreferences(this).getString(MotivationConstants.KEY.USER_NAME)
        binding.textUserName.text = "Olá, $name!"
    }
}